<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharlarVantana</class>
 <widget class="QMainWindow" name="CharlarVantana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CharlarVantana</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(0, 255, 0);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="etiServidor">
          <property name="text">
           <string>Servidor</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ipEditor">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="etiPuerto">
          <property name="text">
           <string>Puerto</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="puertoEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnConectar">
        <property name="text">
         <string>Conectar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDesconectar">
        <property name="text">
         <string>Desconectar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="mensajeVisto"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="textoEditor"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnMandar">
        <property name="text">
         <string>Mandar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
